web/api/health/README.md

```diff
@@ -142,6 +143,43 @@ Example 2.2: Add one more selector, to also silence alarms for cpu1 and cpu2
 http://localhost/api/v1/manage/health?families=cpu1 cpu2
 ```
 
+### List silencers
+
+The command `LIST` was added in netdata v1.13.1 and returns a JSON with the current status of the silencers.
```

Comment:
same here

Reply:
This is completely new and I fixed the documentation with the last commit.

Reply:
This was also fixed with the last commit.